CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.0.7-dev2
+## 0.0.7-dev3
```

Comment:
I know I asked for this change but at this point this is the last PR in flight for today. Can you revert this back to the solid `0.0.7` so we can cut the new release after this is merged?